README.md

```diff
@@ -24,21 +15,121 @@ Its meetings and membership are largely focused on the direction, design, and wo
+ 
+* Localization
+  * Description: Packages to localize maritime vehicles.
+  * Motive: Underwater and surface vehicles have a lot of differences regarding the sensors they can use for localizing themselves. GPS does not work underwater, as well as lidars or electromagnetic waves for long distances... But acoustic waves work pretty well, so underwater localization rely more on sensors like DVL and Sonar, that uses acoustic waves to measure distance and velocity.  
+  * Repositories:
+    * **[Proposal]**: Multilateration of acoustic beacons
+  * References: 
+    *  [Mesh Navigation](https://github.com/uos/mesh_navigation)
+    *  [Fuse Localization](https://github.com/uos/mesh_navigation)
+    *  [MagNav](https://www.gpsworld.com/us-air-force-to-explore-navigating-with-magnetism/)
+
+* Perception
+  * Description: Packages for perception algorithms for maritime robotics.
+  * Motive: Since there are different sensors such as sonars, there is a need to process this data to obtain information. Moreover, it can be created some common structures detection.
+  * Repositories:
+    * **[Proposal]**: Common Maritime Environments Detections (buoys, valves, pipes...)
+    * **[Proposal]**: Fusion of sonar and camera data
+  * References: 
+    * Suggestions...
+   
+* Robots
+  * Description: Packages with maritime robots and structures.
+  * Motive: All the software being developed shall have not only unit tests but integration tests, so robots and structures shall be developed for this purpose. These shall include vcstool for specifying packages to be used with the robot as well as ros_ign_bridge to test it in simulation.
+  * Repositories
+    * **[Proposal]** Hover type UW vehicle
+    * **[Proposal]** Torpedo shaped UW vehicle
+    * **[Proposal]** USV
+    * **[Proposal]** Intervention UW vehicle 
+  * References: 
+    *  Intervention UW sample [RexROV2](https://uuvsimulator.github.io/packages/rexrov2/intro/)
+    *  Torpedo Shaped UW vehicle [LRAUV](https://github.com/osrf/lrauv/)
+   
+* Simulation
+  * Description: Documentation and index for maritime robotics Ignition plugins.
+  * Motive: There is no need to mix ROS code with simulation, also there is already some maritime robotics development in Ignition, so it is a good thing to contribute to ign-gazebo repo instead of creating other in this organization.
+  * Repositories
+    * **[Proposal]** Maritime Robotics Ignition Plugins Index
+    * **[Proposal]** Wave Simulation (repo shall be in ign-plugins)
+    * **[Proposal]** Acoustics Simulation (repo shall be in ign-plugins) (Include salinity and temperature effects)
+    * **[Proposal]** Radar Simulation (repo shall be in ign-plugins)
+    * **[Proposal]** Sonar Simulation (repo shall be in ign-plugins)
+  * References: 
+    *  [ASV Wave Simulator](https://github.com/srmainwaring/asv_wave_sim/tree/feature/fft_waves)
+    *  [ign-gazebo](https://github.com/ignitionrobotics/ign-gazebo)
```

Comment:
```suggestion
    *  [Gazebo Sim](https://github.com/gazebosim/gz-sim)
```

Reply:
Fixed in 37ccbb7c1737ee3fa30bad63e5eae87a980d46f2